### PR TITLE
ci: added `paths-ignore` in tests GHA

### LINKED ISSUE #462: 🚀 Feature request : CI - ignore some files in event trigger of `tests` GHA
### 🔖 Feature description

We should add `paths-ignore` to event triggering config since running tests where it's not needed is a bit unnecessary...like for the .md files

### 🎤 Pitch

```yaml
paths-ignore:
      - '.gitignore'
      - '**.md'
```

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

### PR DESCRIPTION
**Pull request by Ananya2001-an**
## What does this PR do?

Added `paths-ignore` option for running tests GHA only when certain files are changed.

## Related PRs and Issues

closes #462 

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes.

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -3,8 +3,18 @@ name: Tests
...
+          - ".prettierrc"
+          - ".npmrc"
+          - ".gitignore"
     pull_request:
         branches: [main]
+        paths-ignore:
+          - "**/*.md"
+          - ".prettierrc"
+          - ".npmrc"
+          - ".gitignore"
```
Comment (TGlide) [inline .github/workflows/tests.yml:17]: There are some changes we could do to the paths:
- There are more static files we could include, e.g. .env.example, .eslintrc.cjs, .eslintignore
- We could also ignore assets such as images

**Review by TGlide — COMMENTED** [review]
Thank you for the PR 😄

Just one small comment

File: .github/workflows/tests.yml (hunk 1 above)
Comment (Ananya2001-an) [reply .github/workflows/tests.yml:17]: Yeah that sounds good. I will add those too...

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -3,8 +3,26 @@ name: Tests
...
+          - ".env.example"
+          - ".eslintrc.cjs" 
+          - ".eslintignore"
+          - "static/*"
```
Comment (TGlide) [inline .github/workflows/tests.yml:14]: Maybe `static/**/*` instead?

**Comment by TGlide** [comment]
> Hey @TGlide I have done the changes you can review them again :)

Awesome, thank you! LGTM 🎉

**Pull request merged**